embedded-workshop-book/src/installation.md

```diff
@@ -145,46 +145,6 @@ $ cargo install probe-run
 Installed package `probe-run v0.1.8` (..)
 ```
 
-### Workshop specific tools
-
-The [workshop git repository](https://github.com/ferrous-systems/embedded-trainings-2020) contains custom crates that help you with flashing and debugging the workshops. Depending on the workshop you're attending, you need to install all or some of them.
-
-In any case, clone and change into the repository first:
-
-```console
```

Comment:
I'd leave the instructions to clone the repo in there, they'd probably need a different section heading now though